core/src/context.rs

```diff
@@ -17,6 +17,7 @@ use wasmer_wasi::{Pipe, WasiState};
 use parser::ModuleArguments;
 
 use crate::package::PackageImplementation;
+use crate::CoreError::MissingTransform;
```

Comment:
I think that we don't need to bring the enum variant into scope and instead use it as `CoreError::MissingTransform` like we do with all other errors.

Reply:
In my perfect world, all enums would be in scope all the time :( but okay, I'll remove it